### PR TITLE
Add example of maplibre-gl-draw

### PR DESCRIPTION
**Pull request by birkskyum**
Add example using the new [maplibre-gl-draw](https://github.com/birkskyum/maplibre-gl-draw) plugin thats ported to typescript and refactored.

## Launch Checklist

 - [X] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [ ] Briefly describe the changes in this PR.
 - [ ] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [X] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ## main
 
 ### ✨ Features and improvements
-
 - ⚠️ Change the return type of `on` method to return a `Subscription` to allow for easy unsubscribe ([#5080](https://github.com/maplibre/maplibre-gl-js/pull/5080))
+- Add maplibre-gl-draw example ([#5086](https://github.com/maplibre/maplibre-gl-js/pull/5086))
```
Comment (HarelM) [inline CHANGELOG.md:5]: Not sure this is interesting enough...

File: test/examples/mapbox-gl-draw.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <title>Show drawn polygon area</title>
+    <title>Draw polygon with mapbox-gl-draw</title>
```
Comment (HarelM) [inline test/examples/mapbox-gl-draw.html:4]: I would consider replacing this page instead of adding another one.

**Comment by HarelM** [comment]
Looks good! Thanks! 
Added a few minor comments.

File: test/examples/maplibre-gl-draw.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
     <title>Show drawn polygon area</title>
-    <meta property="og:description" content="Use mapbox-gl-draw to draw a polygon and Turf.js to calculate its area in square meters." />
+    <meta property="og:description" content="Use @birkskyum/maplibre-gl-draw to draw a polygon and Turf.js to calculate its area in square meters." />
```
Comment (HarelM) [inline test/examples/maplibre-gl-draw.html:5]: Didn't you want to change the title as well?

**Review by HarelM — APPROVED** [review]
Looks good to me.

**Comment by birkskyum** [comment]
I don't know what action to take for the **codecov/project** ci - it appear to break on main as well.

**Comment by HarelM** [comment]
I'm waiting for codecov to fix it, I have a direct line to their dev team.
Once this is solved, I'll ask Ramya to remove the codecov app and keep the comment as it was before.
It's Ok, don't worry about it.

**Review by HarelM — APPROVED** [review]

**Comment by HarelM** [comment]
You can follow here if that interest you:
https://github.com/codecov/codecov-action/issues/1662

**Pull request merged**